soccer/planning/Path.cpp

```diff
@@ -25,15 +25,15 @@ void Path::draw(SystemState* const state, const QColor& color,
 
     // Get the closest step size to a desired value that is divisible into the
     // duration
-    const float duration = getDuration();
-    const float desiredStep =
-        0.25;  // draw the path by interpolating every x seconds
+    const RJ::Seconds duration = getDuration();
+    const RJ::Seconds desiredStep(
```

Comment:
Can the comment go above this line so the style checker is happy and dosent split it?